app/addons/documents/index-editor/components.react.jsx

```diff
@@ -24,19 +24,21 @@ function(app, FauxtonAPI, React, Stores, Actions, Components, beautifyHelper) {
   var indexEditorStore = Stores.indexEditorStore;
   var getDocUrl = app.helpers.getDocUrl;
 
-  var ToggleButton = React.createClass({
-
-    render: function() {
+  // global component
+  var StyledSelect = React.createClass({
```

Comment:
Nice, are you going to move this somewhere more general so we can is it elsewhere?